site/src/sphinx/server-access-log.rst

```diff
@@ -104,6 +107,28 @@ using :api:`ServerBuilder`. You may use one of the pre-defined log formats.
     sb.accessLogFormat("...log format...");
     ...
 
+You can have different :apiplural:`AccessLogWriter` for :apiplural:`VirtualHost` and :apiplural:`Service`.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.server.logging.AccessLogWriter;
+
+    ServerBuilder sb = new ServerBuilder();
+    sb.virtualHost("foo.com")
+      .accessLogWriter(AccessLogWriter.combined(), true)
+      .service((ctx, req) -> HttpResponse.of(OK)); // This service will log using the combined format.
+      ...
+
+    sb.route()
+      .get("/commonAccessLog")
+      .accessLogWriter(AccessLogWriter.common(), true)
+      .build((ctx, req) -> HttpResponse.of(OK)); // This service will log using the common format.
+
+    AccessLogWriter myWriter = ...
+    sb.accessLogWriter(myWriter);
```

Comment:
- How about renaming `myWriter` to `fallbackLogWriter`?
- How about moving the two sentences to the beginning of the example?

Reply:
Thanks! fixed! :bowing_man: